replybot/lib/typewheels/transition.js

```diff
@@ -63,8 +64,9 @@ class Machine {
                                           pageId, shortcode, startTime)
 
     const user = await this.getUser(userId, pageToken)
+    const surveyMetadata = await getSurveyMetadata(surveyId)
 
-    const actions = act({form, user}, state, output)
+    const actions = act({form, user, surveyMetadata, timestamp}, state, output)
```

Comment:
I actually don't think `act` is the right place to put this. The way the event-sourced state machine works, there is `exec` which does most of the work and creates an intermediate representation called `output`. That `output` is then used to transition to the new state via `apply` and also to generate side effects via `act`. 

But really, when the survey is `off`, the state should really move to an `end` state. So the output will need to be different and `exec` should make the decision.